lib/chef/knife/core/windows_bootstrap_context.rb

```diff
@@ -125,29 +109,29 @@ def config_content
           end
 
           if config[:ssl_verify_mode]
-            client_rb << %Q{ssl_verify_mode :#{config[:ssl_verify_mode]}\n}
+            client_rb << %Q{ssl_verify_mode :#{chef_config[:ssl_verify_mode]}\n}
```

Comment:
shouldn't all the above code somehow inherit from the superclass, because it looks like copypasta?

Reply:
Updated!